db/migrate/20241121050334_add_phone_number_to_people.rb

```diff
@@ -0,0 +1,12 @@
+class AddPhoneNumberToPeople < ActiveRecord::Migration[7.2]
+  def change
+    add_column :people, :phone_number, :string, limit: 15
```

Comment:
Replaced the `phone` column with `phone_number` to match the organizations table naming convention. Plus, I think it is more explicit.

Also added some simple checks to sanitize data.